### DISCUSSION TITLE
Beginner issue for a new contributor

### DISCUSSION THREAD

**Discussion by vorbrodt** [axios/axios#4932] [General]
Hello, I'm a Swedish software engineering student who is looking for an open source project to contribute to and since I have used axios many times in my own projects I feel interested in contributing to it. However, I have some questions before I start investing more time into it.

1. Are there beginner friendly issues that are suitable for new contributors? I saw that there was a label for "beginner", but no one seems to be using it or is there simply no beginner issues available?
2. How is the communication and activity within the community? Is this the main communication channel or is there a discord or messenger group I'm not aware of?
3. How long does it take to get a pull request accepted? I see that there are some pull requests that have been up for a couple of months.

**Comment by jasonsaayman** [comment]
Hi 👋 thanks for reaching out.

1. We have probably been pretty bad at assigning that tag, partly my fault as I have been cleaning issues up and just trying to stay ahead of that is quite a lot of work. However, I will start actively looking into this.
2. There is a slack channel, we only add via invite however a discord app is something we could look into if there is interest.
3. This depends, some of the pull requests have stayed open for a while pending the release of v1. There is also some issue with people taking very long to reply if there are questions but a small backlog has grown again and I will take care of that as soon as possible.

If you would like we can chat further just email me and I will invite you on slack.

**Reply by tenebra-dev** [reply]
Hey @jasonsaayman , I'm in the same boat, I would love to join a Discord Server or Slack to catch up with the project. I want to start contributing aswell but I need to understand it properly first, just a place where I can ask questions, see people talking more about it really would help, thank you.

**Comment by Abhiram716** [comment]
Hey @jasonsaayman I am looking to take up on any good first issues too can you point me if there are any available.
